pyproject.toml

```diff
@@ -1,16 +1,30 @@
 [tool.poetry]
 name = "las-geoh5"
-version = "0.0.1"
+version = "0.1.0"
 description = "Las/Geoh5 conversion"
 authors = ["Mira Geoscience <support@mirageoscience.com>"]
+keywords = ["geology", "geophysics", "earth sciences"]
```

Comment:
must set the license type, and point to the README file

For license keyword to use for MIT, see https://python-poetry.org/docs/pyproject/#license

And no need to place the license in classifiers: poetry will take care of that

Reply:
add homepage as well (see pyproject in geoh5py)

Reply:
and doner